be/src/formats/parquet/file_writer.cpp

```diff
@@ -194,140 +305,35 @@ Status FileWriterBase::init() {
     return Status::OK();
 }
 
-::parquet::RowGroupWriter* FileWriterBase::_get_rg_writer() {
-    if (_rg_writer == nullptr) {
-        _rg_writer = _writer->AppendBufferedRowGroup();
+void FileWriterBase::_generate_chunk_writer() {
+    if (_chunk_writer == nullptr) {
```

Comment:
If you want to use ptr=null, you need to use pointer. If you want to use unique_ptr, I think it's better to use reset()

Reply:
This line is to check nullity